From 16709de672bbb91256549e9b11b962b850d71754 Mon Sep 17 00:00:00 2001
From: "Subhransu S. Prusty" <subhransu.s.prusty@intel.com>
Date: Thu, 5 Jun 2014 14:11:06 +0530
Subject: ASoC: codecs: Cache sync not required for rt5640

As the codec is not reset during suspend/resume, cache sync is not
required. Even if we need to reset the codec, cache sync can go in
resume.

Change-Id: Ieaeeb66e80fd62612ca11d8073334fe29aacf571
Signed-off-by: Subhransu S. Prusty <subhransu.s.prusty@intel.com>
---
 sound/soc/codecs/rt5640.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/soc/codecs/rt5640.c b/sound/soc/codecs/rt5640.c
index 46e647c..8809f8c 100644
--- a/sound/soc/codecs/rt5640.c
+++ b/sound/soc/codecs/rt5640.c
@@ -3175,7 +3175,6 @@ static int rt5640_set_bias_level(struct snd_soc_codec *codec,
 #endif
 			codec->cache_only = false;
 			codec->cache_sync = 1;
-			snd_soc_cache_sync(codec);
 			rt5640_index_sync(codec);
 		}
 		break;
-- 
2.0.0

